README.md

```diff
@@ -32,6 +32,14 @@ Administration => Roles & Permissions
     cd assets.src && npm install
     rake provision
```

Comment:
This doesn't work like this, right? Isn't this to be executed inside of the running container?